Skyrat Floor Icon Rework + Box Touchups

R0bro opened the pull request:
## About The Pull Request

I was initially going to put my wall and tile changes into one big pr, but I've decided to chop them in half here for the sake of getting things out there. This pr primarily aims at reworking Skyrat's tile replacements to fit more in line with TG's spriting style.

Update: I've taken the liberty of modifying box a bit since I had to go in and change some assets that were directly pointing to the old Skyrat floors dmi. The intention of the additional changes is to simplify some elements, unify the theming and increase visual clarity.

## Why It's Good For The Game

Skyrat tiles have this style to them, a very specific style that goes against modern TG spriting sensibilities. The choice to use textured floors in place of smooth ones has always bothered me. They are visually noisy and I believe it is a reason why maps like nebula often feel overwhelming to look at. I aimed to make something in-between, at least palette wise. The basic floor tiles are ever so slightly softer in color than tg's, so as to not make them overbearing to look at.

## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

<img width="861" height="451" alt="dreamseeker_Q8H2khkTGH" src="https://github.com/user-attachments/assets/5989e4c3-5b05-4950-b7a1-ff79334c763e" />

<img width="750" height="830" alt="dreamseeker_fuQVxuJfhv" src="https://github.com/user-attachments/assets/d695f785-58c2-41c5-afb7-66a7560f6b6e" />

<img width="649" height="514" alt="dreamseeker_dEkpDV5etW" src="https://github.com/user-attachments/assets/35daf939-c9c1-472a-92b7-1c3cacaa6e04" />


</details>

## Changelog

:cl:
image: updated/reworked many Skyrat and TG floor icons
image: various shuttle tiles now have associated stack objects
map: updated box station visuals
map: updated icebox persistence roof visuals
/:cl:

R0bro committed:
Bye bye skyrat floors. You may or may not be missed.

R0bro committed:
Swapped last remaining direct references to the now removed "modular_skyrat/modules/aesthetics/floors/icons/floors.dmi"

R0bro committed:
Shuttle floors! Also those hideous pool tiles...

R0bro committed:
Forgot the living floor. :P

R0bro committed:
I don't know who had the idea to make a few tile point directly to the Skyrat floors dmi but I really hope nobody does something similar in the future.

Also took the liberty of tidying up the visuals.

R0bro committed:
gwuh

R0bro committed:
Despite transferring the dmi for the catwalk plating to it's new folder, I forgot to actually modify it. Whoops!

I forgot the cargo shuttle floors too!.
Furthermore, someone asked to put a camera in the mining dock so sci can see the boulder machine, so I did that on box.

shayoki commented:
I am 100% in favor of this. Skyrat tiles and wallstuffs need to go in exchange to this. Keep it up! ✨️

R0bro committed:
Hi I have never resolved a conflict before so I may be stupid.

R0bro committed:
Merge branch 'Bubberstation:master' into Floor-resprites

R0bro committed:
Now are we good?

R0bro committed:
Merge branch 'Floor-resprites' of https://github.com/R0bro/Bubberstation into Floor-resprites

# Conflicts:
#	modular_skyrat/modules/aesthetics/floors/code/floors.dm

R0bro committed:
Whoops again. I accidentally replaced a line with the wrong thing.

R0bro committed:
I cry.

R0bro committed:
I cry even harder.

R0bro committed:
Whoops, I tied the cargo cam console to look for the vault but it failed.

R0bro committed:
Somehow I did not fix the network with the last commit. NOW it's good.

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Since this pr is still up, I resprited the hazard stripe decals and recolored the broken tiles because I missed those before.

R0bro committed:
I forgot a whole 9 pixels (how dare they do this).

R0bro committed:
Merge branch 'master' into Floor-resprites

xPokee reviewed (approved):
The sprites look fantastic and this is fully modular. Great work!

Nuk5513 commented:
those are literally floor sprites from 7 years ago

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Touched up a few more areas and things I missed on box.

Adjusted the siding decals a tiny bit so they fit in better.

lessthnthree committed:
Merge remote-tracking branch 'upstream/master' into pr/4918

lessthnthree committed:
Update BoxStation.dmm

lessthnthree committed:
Revert "Update BoxStation.dmm"

This reverts commit c4d311782719220b42329a71e669b39391e7c556.

lessthnthree committed:
Update BoxStation.dmm

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Destroyed tile variants and the cult tiles now are properly replaced.

A few minor tweaks on box, namely that one 'security locker' in maintenance looks convincing now.

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Moves the decal replacements into one dmi. I would prefer not to do it but for future purposes it's better this way.

R0bro committed:
Merge branch 'Floor-resprites' of https://github.com/R0bro/Bubberstation into Floor-resprites

R0bro commented:
> are the glass material floors meant to no longer be transparent?

That is not intentional. I will have to look.

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Minor box tweaks, mainly fixing a missing cable in maints.

The cargo shuttle tiles have a new state for mappers (I really should just move them to the zubbers folder at this rate).

R0bro committed:
Accidentally messed up an icon state whoops, fixed. 🤤

R0bro committed:
Merge branch 'master' into Floor-resprites

lessthnthree committed:
Merge remote-tracking branch 'upstream/master' into pr/4918

lessthnthree committed:
Merge branch 'Floor-resprites' of https://github.com/R0bro/Bubberstation into pr/4918

shayoki committed:
Update BoxStation.dmm

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Fixed access on the labor shuttle airlock (thanks Gavla)

R0bro committed:
I was blind (sorry Gavla)

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Rearranged the coffee store and auxiliary storage as well as fixing some missing scrubber pipes under a door or two.

R0bro committed:
Forgot a camera, whoops.

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Whoops, forgot fire safety (alarms)!

R0bro committed:
My brother in Christ you forgot QUOTES!

R0bro committed:
(you still forgot them)

R0bro committed:
"Please upper-case your door names." Uh nuh uh (does it anyway).

lessthnthree committed:
Merge remote-tracking branch 'upstream/master' into pr/4918

lessthnthree committed:
Merge branch 'Floor-resprites' of https://github.com/R0bro/Bubberstation into pr/4918

R0bro committed:
Fixed my erroneous area designation that I forgot about (even though I thought I already did).

R0bro committed:
I am tired

R0bro committed:
(they were tired)

R0bro committed:
face falls off*

lessthnthree commented:
These tiles could be improved by not being segmented into 4

<img width="200" height="231" alt="image" src="https://github.com/user-attachments/assets/b9adf632-3b47-4a2d-a755-ecaa6b0dbf22" />

lessthnthree committed:
Merge remote-tracking branch 'upstream/master' into pr/4918

lessthnthree committed:
missing door buttons

R0bro committed:
Merge branch 'master' into Floor-resprites

lessthnthree committed:
Merge remote-tracking branch 'upstream/master' into pr/4918

lessthnthree committed:
punch line

lessthnthree committed:
decal fix

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Tile stack sprites now more closely match the replacements.

Reworked the weird height floors from skyrat that didn't look even close to what they were meant to represent.
A few visual errors corrected on box. A decal/tile or two.

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Merge branch 'master' into Floor-resprites

R0bro committed:
Various skyrat shuttle tiles can now be placed by aspiring carpenters in game.

R0bro committed:
Merge branch 'master' into Floor-resprites

shayoki committed:
Merge branch 'master' into pr/4918
